src/main/java/LacksDataBaseProject/User.java

```diff
@@ -0,0 +1,66 @@
+package LacksDataBaseProject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class User extends LacksOperations {
+    protected String userName;
+    protected String login;
+    protected String password;
+    protected String skypeID;
+    protected Role role;
+
+    AddAndRemoveSupplier addAndRemoveSupplier;
```

Comment:
brakuje porządku w deklaracji pól, może lepiej by było by wszystkie pola z tym samym modyfikatorem dostępu trzymać w jednym miejscu, w sensie grupować :) bo terazn macie pola które łamią tą zasadę i też trochę czytelność, ale to raczej już gwestia gustu :)